### PR TITLE
Use aws ssm parameter for referencing port number

### PR DESCRIPTION
**Pull request by carminetoriello**
## What 🕵️ 🔍

- Uses the aws ssm parameter DE_PORT that has been created on aws for referencing the port number within terraform file

## Reviewer checklist

*Reviewer agreement:*

* Reviewers shouldn't review a "Draft Pull Request", but rather wait for it to be marked as "Ready for Review".
* Reviewers do **not** commit or merge the pull request.
* If/when there are changes that need to be applied (applying Terraform or deploying Kubernetes manifests), the **assignee** is the one responsible for doing it.
* Use the labels `not-applied`, `partially-applied` and `fully-applied` so others know the current progress.
  * Remember to change labels accordingly once you start and finish to apply the changes.

### CODE REVIEW

**Review by jorge-cr-13 — APPROVED** [review]
LGTM

File: aws/stage/ecs/main.tf [hunk 1]
```diff
@@ -16,6 +16,10 @@ data "terraform_remote_state" "load_balancer" {
   }
 }
 
+data "aws_ssm_parameter" "de_port" {
+  name = "DE_PORT"
```
Comment (jorge-cr-13) [inline aws/stage/ecs/main.tf:20]: What does `DE_PORT` stands for?

Comment (carminetoriello) [reply aws/stage/ecs/main.tf:20]: I've taken it from the parameters listed in the task linked to this one. Here is the task https://xainag.atlassian.net/browse/ET-3366

Comment (jorge-cr-13) [reply aws/stage/ecs/main.tf:20]: Ok thanks!

**Comment by jorge-cr-13** [comment]
I think is better if we close this PR and manage all these changes under this [PR](https://github.com/xaynetwork/xayn_infrastructure/pull/677)

**Comment by carminetoriello** [comment]
Sure, will close this one!

**Pull request closed without merging**